Oxford Reference.js

```diff
@@ -0,0 +1,226 @@
+	var rowsExtendedTitle = ZU.xpath(doc, '//span[@class="title"]');
+	for (var i=0; i<rows.length; i++) {
+		var href = rows[i].href;
+		var title = ZU.trimInternal(rows[i].textContent) + ',' + ZU.trimInternal(rowsExtendedTitle[i].textContent);
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[href] = title;
+	}
+	return found ? items : false;
+}
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
+			if (!items) {
+				return true;
+			}
+			var articles = [];
+			for (var i in items) {
+				articles.push(i);
+			}
+			ZU.processDocuments(articles, scrape);
+		});
+	} else {
+		scrape(doc, url);
+	}
+}
+
+function scrape(doc, url)
+{
+	if (detectWeb(doc, url) == "book")
+		var item = new Zotero.Item("book");
+	else
+		var item = new Zotero.Item("bookSection");
+
+	var edition = ZU.xpathText(doc, '//meta[@property="http://schema.org/bookEdition"]/@content');
+	if(edition)
+		item.edition = edition;
+
+	var dateCreated = ZU.xpathText(doc, '//meta[@property="http://schema.org/dateCreated"]/@content');
+	if(dateCreated)
+		item.date = dateCreated;
+	else
+		item.date = ZU.xpathText(doc, '//dl[@class="metadata metadataPrintPublicationDate"]/dd');
+	
+	var isbn = ZU.xpathText(doc, '//meta[@property="http://schema.org/isbn"]/@content');
+	if(isbn)
+		item.ISBN = isbn;
+	else
+		item.ISBN = ZU.xpathText(doc, '//dl[@class="metadata metadataPrintIsbn13"]/dd');
+	
+	var publisher = ZU.xpathText(doc, '//meta[@property="http://schema.org/publisher"]/@content');
+	if(publisher)
+		item.publisher = publisher;
+	else
+		item.publisher = ZU.xpathText(doc, '//dl[@class="metadata metadataPublisher"]/dd');
+	
+
+	var title = ZU.xpathText(doc, '//meta[@property="og:title"]/@content');
+	if(title)
+		item.title = title;
+	else
+		item.title = ZU.xpathText(doc, '//h1[@id="pagetitle"]');
+	
+	var abstract = ZU.xpathText(doc, '//meta[@name="description"]/@content');
+	if(abstract)
+		item.abstractNote = abstract;
+	else
+		item.abstractNote = ZU.xpathText(doc, '//div[@class="abstract"]');
+	
+	var editors = ZU.xpath(doc, '//meta[@property="http://schema.org/editor"]/@content');
+	if(editors)
+	{
+		for (var i in editors){
+			item.creators.push(ZU.cleanAuthor(editors[i].textContent,"editor"));
+		}
+	}
+	
+	var authors = ZU.xpathText(doc,'//dl[@class="metaInfo"]/dd[@class="author"]');
+	if(authors)
+	{
+		authors = authors.split(',');
+		for (var i in authors){
+			item.creators.push(ZU.cleanAuthor(authors[i], "author"));
+		}
+		
+	}
+	
+	item.attachments = ({
+		url: url,
+		title: "Oxford Reference Snapshot",
+		mimeType: "text/html"
+	});
+	
+	item.complete();	
+}/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "http://www.oxfordreference.com/view/10.1093/acref/9780199546572.001.0001/acref-9780199546572-e-0009",
+		"items": [
+			{
+				"itemType": "bookSection",
```

Comment:
For chapters we should also save the title of the book. This seems missing here.